webviz_subsurface/plugins/_surface_viewer_fmu.py

```diff
@@ -10,6 +10,7 @@
 from dash.exceptions import PreventUpdate
 from webviz_config import WebvizPluginABC, WebvizSettings
 from webviz_config.webviz_store import webvizstore
+from webviz_config.deprecation_decorators import deprecated_plugin
```

Comment:
The import order linting is complaining about this one. Reorder and then we merge? 🎉